vms/proposervm/block.go

```diff
@@ -401,7 +401,7 @@ func (p *postForkCommonComponents) shouldBuildBlockPostDurango(
 	parentTimestamp time.Time,
 	parentPChainHeight uint64,
 	newTimestamp time.Time,
-) error {
+) (bool, error) {
```

Comment:
flipped the logic to align with `verifyPre/PostDurangoBlockDelay`. Signed blocks will be the new normal so I chose to signal "should build signed block/should have a proposer" with true